.github/workflows/test.yml

```diff
@@ -30,7 +30,7 @@ jobs:
   pwa-kit:
     strategy:
       matrix:
-        node: [14]
+        node: [14, 16]
         npm: [6, 7, 8]
     runs-on: ubuntu-latest
     env:
```

Comment:
Since Node 16 is our active version, we should also change the default npm to npm@8.
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/0748de107d9e86ce74d0fa0684369ede98e7810a/.github/workflows/test.yml#L36-L40